### PR TITLE
Minor doc enhancements to Identity providers

### PR DESCRIPTION
**Pull request by michelle-purcell**
To enhance and apply style as per the [Quarkus docs contributor style guide](https://quarkus.io/guides/doc-reference) and some prep for future 3.2 product docs.

### CODE REVIEW

File: docs/src/main/asciidoc/security-identity-providers.adoc [hunk 1]
```diff
@@ -9,18 +8,18 @@ include::_attributes.adoc[]
 :diataxis-type: concept
 :categories: security
 
-In the Quarkus Security framework, identity providers play a key role in authentication and authorization, providing services for storing and verifying user identities.
-The Jakarta Persistence `IdentityProvider` creates a `SecurityIdentity` instance used during user authentication to verify and authorize access requests, making your Quarkus application secure.
+In the Quarkus Security framework, identity providers play a crucial role in authentication and authorization, providing services for storing and verifying user identities.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-identity-providers.adoc:11]:
```suggestion
In the Quarkus Security framework, identity providers play a crucial role in authentication and authorization, providing services for storing and verifying user identities.
```

File: docs/src/main/asciidoc/security-identity-providers.adoc [hunk 2]
```diff
@@ -9,18 +8,18 @@ include::_attributes.adoc[]
...
+The Jakarta Persistence `IdentityProvider` creates a `SecurityIdentity` instance, which gets used during user authentication to verify and authorize access requests, making your Quarkus application more secure.
```
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-identity-providers.adoc:12]: I know this PR is unrelated to it :-), but these 2 sentences need to be updated as they are not quite correct, let me try

File: docs/src/main/asciidoc/security-identity-providers.adoc (hunk 1 above)
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-identity-providers.adoc:11]:
```suggestion
In the Quarkus Security framework, identity providers play a crucial role in authentication and authorization by  verifying user identities.
```

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-identity-providers.adoc:11]: `providing services for storing and verifying user identities` does not sound quite right, one of the identity providers uses the internally stored identities to support their verification but at the Quarkus security architecture level, it is an implementation detail of the (JPA) identity provider

File: docs/src/main/asciidoc/security-identity-providers.adoc (hunk 2 above)
Comment (sberyozkin) [inline docs/src/main/asciidoc/security-identity-providers.adoc:12]:
```suggestion
`IdentityProvider` creates a `SecurityIdentity` instance, which gets used during user authentication to verify and authorize access requests to your Quarkus application.
```

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-identity-providers.adoc:12]: `The Jakarta Persistence` should not be referred to in the Introduction, it is an individual provider which uses JPA and it is linked to below

Comment (sberyozkin) [reply docs/src/main/asciidoc/security-identity-providers.adoc:12]: Also, there could be only the secure access allowed if the verification/authorization has passed, so may be we can just drop this last clarification as `uring user authentication to verify and authorize access requests` implies it

File: docs/src/main/asciidoc/security-identity-providers.adoc (hunk 1 above)
Comment (michelle-purcell) [reply docs/src/main/asciidoc/security-identity-providers.adoc]: @sberyozkin - Fixed. Thanks.

File: docs/src/main/asciidoc/security-identity-providers.adoc (hunk 2 above)
Comment (michelle-purcell) [reply docs/src/main/asciidoc/security-identity-providers.adoc]: Thanks for your suggestion below :point_down:

**Comment by michelle-purcell** [comment]
@sberyozkin - Thanks for your review and updates. It's great to catch these items and fix them in here before the 3.2 doc publish, especially as they do not cause PR scope creep.
Did you have others to discuss in this PR or are we good to merge? Thanks

**Comment by sberyozkin** [comment]
@michelle-purcell thanks, sure we can wait till tomorrow, this PR is quite small though,  @sheilamjones, how does it look to you?

**Comment by sberyozkin** [comment]
@michelle-purcell Oh sorry, I think you asked if I had other items to discuss in mind. No, LGTM, @sheilamjones, hope you don't mind me merging, it is a small PR and it will help 2 other PRs from Michelle go ahead, thanks

**Review by sberyozkin — APPROVED** [review]

**Comment by michelle-purcell** [comment]
Thanks @sberyozkin :sun_with_face:

**Pull request merged**